src/Vue/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "symfony/ux-vue",
+    "type": "symfony-bundle",
+    "description": "Integration of Vue.js in Symfony",
+    "keywords": [
+        "symfony-ux"
+    ],
+    "homepage": "https://symfony.com",
```

Comment:
Can it be the ux.symfony.com ?
(I do not know the standards)

Reply:
This is the same for all UX components at the moment. Changing this would require to update all components and not sure it's wanted